libs/labelbox/pyproject.toml

```diff
@@ -87,9 +89,8 @@ unit = "pytest tests/unit"
 # LABELBOX_TEST_BASE_URL="http://host.docker.internal:8080" \
 integration = { cmd = "pytest tests/integration" }
 data = { cmd = "pytest tests/data" }
-yapf-lint = "yapf tests src -i --verbose --recursive --parallel --style \"google\""
 mypy-lint = "mypy src --pretty --show-error-codes --non-interactive --install-types"
-lint = { chain = ["yapf-lint", "mypy-lint"] }
```

Comment:
Should we add fmt to the chain here?

Reply:
Yea added this and removed my extra workflow job